client/src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ChangeEvent, FormEvent, useCallback, useState } from "react";
+import { GrLogin } from "react-icons/gr";
+import { AiOutlineMail } from "react-icons/ai";
+import { StyledInput } from "./StyledInput/StyledInput";
```

Comment:
import order should be
library
component
hooks
utils
constants
types
and add new line between them to seperate